src/Microsoft.ML.TorchSharp/NasBert/TextClassificationTrainer.cs

```diff
@@ -599,8 +617,13 @@ public SchemaShape GetOutputSchema(SchemaShape inputSchema)
             metadata.Add(new SchemaShape.Column(AnnotationUtils.Kinds.KeyValues, SchemaShape.Column.VectorKind.Vector,
                 TextDataViewType.Instance, false));
 
-            outColumns[_options.OutputColumnName] = new SchemaShape.Column(_options.OutputColumnName, SchemaShape.Column.VectorKind.Scalar,
+            // Get label column for score column annotations. Already verified it exists.
```

Comment:
I didn't think it was needed because `CheckInputSchema` on line 613 will throw an error if it doesn't exist. Do you think I should still assert it here?